src/main/java/kitchenpos/common/Price.java

```diff
@@ -0,0 +1,72 @@
+package kitchenpos.common;
+
+import javax.persistence.Embeddable;
+import java.math.BigDecimal;
+import java.util.Objects;
+
+@Embeddable
+public class Price {
+    private BigDecimal price;
+
+    protected Price() {}
+
+    private Price(final BigDecimal price) {
+        validate(price);
+        this.price = price;
+    }
+
+    public static Price of(int price) {
+        return new Price(BigDecimal.valueOf(price));
+    }
+
+    public static Price of(long price) {
+        return new Price(BigDecimal.valueOf(price));
+    }
+
+    public static Price of(BigDecimal price) {
+        return new Price(price);
+    }
+
+    public Price add(Price other) {
+        return new Price(this.price.add(other.price));
+    }
+
+    public Price subtract(Price other) {
+        return new Price(this.price.subtract(other.get()));
+    }
+
+    public Price multiply(long count) {
+        return new Price(this.price.multiply(BigDecimal.valueOf(count)));
+    }
+
+    public BigDecimal get() {
+        return this.price;
+    }
+
+    public int intValue() {
+        return this.price.intValue();
+    }
+
+    public boolean isBiggerThan(Price other) {
+        return price.intValue() > other.get().intValue();
```

Comment:
큰 차이는 없지만, 인스턴스가 달라도 같은 객체 내에선 필드 접근이 가능하기 때문에 아래와 같이 사용할 수도 있어요~!

```suggestion
        return price.intValue() > other.price.intValue();
```

Reply:
그렇군요 ㅎㅎ

이후에는 필드접근을 활용해보겠습니다! 😁